runtime/doc/options.txt

```diff
@@ -4878,9 +4878,12 @@ A jump table for the options with a short description can be found at |Q_op|.
 	   ordering.  This is for preferences to overrule or add to the 
 	   distributed defaults or system-wide settings (rarely needed).
 
-	More entries are added when using |packages|.  If it gets very long
-	then `:set rtp` will be truncated, use `:echo &rtp` to see the full
-	string.
+							*rtp-packages*
+	"start" packages will additionally be used to search for runtime files
+	after these, but package entries are not visible in `:set rtp`, as the
```

Comment:
I guess they also aren't shown by `:echo &rtp` ? Should we go further and say something like:

> :runtime searches "start" packages, thus they are _logically_ but not explicitly part of 'runtimepath'.